app/views/contexts/summary/question.py

```diff
@@ -95,24 +108,91 @@ def _build_answers(
+        if self.dynamic_answer_schemas:
+            for answer_schema in self.answer_schemas:
+                self.list_item_id = None
+                answer_value: Optional[AnswerValueEscapedTypes] = self.get_answer(
+                    answer_store, answer_schema["id"]
+                )
+                answer = self._build_answer(
+                    answer_store, question_schema, answer_schema, answer_value
+                )
+
+                summary_answer = Answer(
+                    answer_schema=answer_schema,
+                    answer_value=answer,
+                    block_id=block_id,
+                    list_name=list_name,
+                    list_item_id=self.list_item_id,
+                    return_to=return_to,
+                    return_to_block_id=return_to_block_id,
+                ).serialize()
+                summary_answers.append(summary_answer)
+
+            for answer_schema in self.dynamic_answer_schemas:
+                for list_item_id in self.list_store.get(
+                    self.dynamic_answer_identifier
+                ).items:
+                    placeholder_renderer = PlaceholderRenderer(
+                        answer_store=self.answer_store,
+                        list_store=self.list_store,
+                        progress_store=self.progress_store,
+                        schema=self.schema,
+                        language="en",
+                        metadata=None,
+                        response_metadata={},
+                    )
+                    self.list_item_id = list_item_id
+                    dynamic_answer_value: Optional[
+                        AnswerValueEscapedTypes
+                    ] = self.get_answer(answer_store, answer_schema["id"])
+                    answer = self._build_answer(
+                        answer_store,
+                        question_schema,
+                        answer_schema,
+                        dynamic_answer_value,
+                    )
+
+                    resolved_answer_schema = self.schema.get_mutable_deepcopy(
+                        answer_schema
+                    )
+                    resolved_answer_schema["id"] = (
+                        answer_schema["id"] + f"-{self.list_item_id}"
+                    )
+                    rendered_schema = placeholder_renderer.render(
+                        data_to_render=resolved_answer_schema,
+                        list_item_id=self.list_item_id,
+                    )
+
+                    summary_answer = Answer(
+                        answer_schema=rendered_schema,
+                        answer_value=answer,
+                        block_id=block_id,
+                        list_name=list_name,
+                        list_item_id=self.list_item_id,
+                        return_to=return_to,
+                        return_to_block_id=return_to_block_id,
+                    ).serialize()
```

Comment:
The change links do not work correct for dynamic answers. They need to be updated and tested, i.e when you click change, the right item is focused.

Reply:
From what I can see in schema the Change button click focuses on the right answer?